storage/src/adb/store/mod.rs

```diff
@@ -1,4 +1,5 @@
-//! A mutable key-value database that supports variable-sized values.
+//! A mutable key-value database that supports variable-sized values, but without authentication
```

Comment:
nit: `but without authentication.` (think "capabilities" is implicit)

Reply:
done